Mathlib/Logic/Small/Basic.lean

```diff
@@ -57,7 +57,7 @@ theorem small_of_injective_of_exists {α : Type v} {β : Type w} {γ : Type v'}
     infer_instance
 
 /-!
-We don't define `small_of_fintype` or `small_of_countable` in this file,
+We don't define `Finite.toSmall` or `small_of_countable` in this file,
 to keep imports to `Logic` to a minimum.
```

Comment:
Can you delete `small_of_countable`? It's completely superseded now
```suggestion
We don't define `Finite.toSmall` in this file to keep imports to `Logic` to a minimum.
```

Reply:
Sorry, but superseded by what?

Reply:
Ah sorry, I meant the other way around. Can you try deleting `Finite.toSmall` and renaming `small_of_countable` to `Countable.toSmall`?

Reply:
Done.